foursight_core/react/api/react_ui.py

```diff
@@ -0,0 +1,103 @@
+import os
+import io
+
+REACT_BASE_DIR = os.path.join(os.path.dirname(__file__), "../../react/ui")
+REACT_DEFAULT_FILE = "index.html"
+REACT_WHITELISTED_FILE_PATH_SUFFIXES = [
+    "/index.html",
+    "/main.js",
+    "/main.css",
+    "/manifest.json",
+    "/asset-manifest.json",
+    ".jpeg",
+    ".jpg",
+    ".png",
+    ".ico"
+]
+
+
+class ReactUi:
+
+    def __init__(self, react_api):
+        self.react_api = react_api
+
+    class Cache:
+        static_files = {}
```

Comment:
Is this just a convention you're sticking to across different classes? It makes sense in theory but shouldn't it be an attribute from a centralized class? It doesn't seem like you do much with this except access the elements by name, so it's not clear to me what's gained here by using the class.